examples/config-bench.json

```diff
@@ -10,6 +10,6 @@
     },
     "log": {
         "level": "debug",
-        "output": "/dev/null"
```

Comment:
The idea behind doing /dev/null was to make sure that logging isn't affecting performance. Revert these config example changes?